### ISSUE TITLE
Problem with multiple rules (allow then disallow)

### ISSUE THREAD

**Issue by hbea2014** [t1gor/Robots.txt-Parser-Class#30]
I was testing your parser and I noticed this:

```php
<?php

require __DIR__ . '/vendor/autoload.php';

$parser = new RobotsTxtParser('
User-agent: *
Allow: /
Disallow: /private/
Disallow: /secret/page.html
');

var_dump(
    $parser->isAllowed('/lalala'), // returns false instead of true
    $parser->isAllowed('/public/page.html'), // returns false instead of true
    $parser->isDisallowed('/lalala'), // returns false as it should
    $parser->isDisallowed('/public/page.html') // returns false instead of true
);

```

**Comment by JanPetterMG** [comment]
I tested the exact same code, and I got a different result.
````
$parser->isAllowed('/lalala'), // bool(true)
$parser->isAllowed('/public/page.html'), // bool(true)
$parser->isDisallowed('/lalala'), // bool(false)
$parser->isDisallowed('/public/page.html') // bool(false)
````

Are you sure you are using the most recent version? Changes were committed about 2-3 weeks ago, with lots of bugs fixed...

Try downloading the source php file directly, the zip files in the release section are outdated...
https://github.com/t1gor/Robots.txt-Parser-Class/blob/master/source/robotstxtparser.php
Alternatively download this zip file instead, containing all the latest changes: https://github.com/t1gor/Robots.txt-Parser-Class/archive/master.zip
Preferred, if you are using composer, get the ``dev-master`` version.

Maybe @t1gor could release the current code as a bugfix release? There is still issues to be resolved but it is definitely more stable than the "latest" version released a half year ago...
